The x86 version of the MINGW compiler does not correctly define NULL in some
modes of C++ compilation. Override the definition of NULL, using the safe
default "0".

--ajs

Index: libmodplug/src/libmodplug/stdafx.h
===================================================================
--- libmodplug/src/libmodplug/stdafx.h	(revision 326)
+++ libmodplug/src/libmodplug/stdafx.h	(working copy)
@@ -79,6 +79,12 @@
 
 #endif // !__WIN32__
 
+// Hack for MINGW x64 compiler
+#ifdef NULL
+#undef NULL
+#define NULL 0
+#endif
+
 inline LONG MulDiv (long a, long b, long c)
 {
   // if (!c) return 0;
